homeassistant/components/renault/manifest.json

```diff
@@ -1,6 +1,7 @@
 {
   "domain": "renault",
   "name": "Renault",
+  "integration_type": "hub",
```

Comment:
I think most users have a single Renault vehicle, but the integration is happy to support multiple vehicles within the same account.